toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/RestUtils.java

```diff
@@ -241,6 +263,15 @@ private static HttpPost buildJsonHttpPost(HttpRequest requestInfo) throws Unsupp
 
 	private static CloseableHttpClient buildClient(final HttpRequest requestInfo) {
 		HttpClientBuilder httpBuilder = HttpClients.custom();
+        try {
+			PoolingHttpClientConnectionManager cm = setHttpConnectionManager();
+			httpBuilder.setConnectionManager(cm);
+		} catch (KeyManagementException | NoSuchAlgorithmException
+				| KeyStoreException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)